manifests/state-ofed-driver/0050_ofed-driver-ds.yaml

```diff
@@ -143,6 +143,21 @@ spec:
             {{- end }}
           {{- end }}
           {{- end }}
+          lifecycle:
+            postStart:
+              exec:
+                command:
+                  - /bin/sh
+                  - -c
+                  - |
+                    echo "[INFO] Updating system CA certificates..."; \
```

Comment:
Can you explain what is the issue you want to fix?
- In which flow it happens? Is it only if user configure additional 'certConfig' mount?
- What are the errors we get?
- Why not add this handling in [entrypoint](https://github.com/Mellanox/doca-driver-build/blob/main/entrypoint.sh) ?
- Will it work in disconnected env?
- We support also SLES, what it the relevant command?
- What happen if 'post-start' fails or time out?
- Why it the path change for Ubuntu needed? '`/usr/local/share/ca-certificates`

Reply:
@rollandf  Thanks for the suggestion!
I’ve updated this PR to:
- Fix the certificate path to /usr/local/share/ca-certificates

- Move the update-ca-certificates logic into the entrypoint.sh for Ubuntu-based containers PR-https://github.com/Mellanox/doca-driver-build/pull/104

- SLES support and update-ca-trust handling will be addressed in a follow-up PR.

Let me know if anything further needs adjustment!